src/tests/utils/ApiTestSteps.ts

```diff
@@ -294,3 +294,11 @@ export async function validateTxMAEventData(keyList: any): Promise<any> {
 			});
 	}
 }
+
+export async function validateCriStartTxMAEvent(key: any, context: string): Promise<any> {
+		const getObjectResponse = await HARNESS_API_INSTANCE.get("/object/" + key, {});
+		if (getObjectResponse.data.event_name === "CIC_CRI_START") {
+			console.log(JSON.stringify(getObjectResponse.data, null, 2));
+			expect(getObjectResponse.data.extensions.evidence.context).toBe(context);
```

Comment:
We won't even have the `extension` object if the context isn't `bank_account` so I'm not totally sure how this is passing?

Reply:
Sorry I'm confused though the new test added runs 
```
it.each([
		["NO_PHOTO_ID", "bank_account"],
	])
```

How is this passing for `NO_PHOTO_ID`?

Reply:
A journey type of NO_PHOTO_ID sets the context to bank_account when making the call to start the stub 

<img width="691" alt="image" src="https://github.com/govuk-one-login/ipv-cri-cic-api/assets/110032361/5c157882-2c3d-4421-8506-9dd2ecaa0683">

Reply:
oh! So we are doing bank account twice here?

Reply:
Updates made to change test data from array to object